users/models.py

```diff
@@ -23,12 +25,20 @@ class UsersGroups(models.Model):
     name = models.CharField(max_length=64)
 
 
-class Users(models.Model):
+class Users(AbstractUser):
+    username = None
+    is_staff = None
+    last_login = None
+    date_joined = None
+
     first_name = models.CharField(max_length=64)
     last_name = models.CharField(max_length=64)
-    email = models.CharField(max_length=128)
+    email = models.EmailField(_('email address'), unique=True)
+
     title = models.CharField(max_length=32,
                              choices=[(tag, tag.value) for tag in AcademicTitle], null=True)
-    password_hash = models.CharField(max_length=64)
```

Comment:
Why have you removed `password_hash`? Shouldn't we store this to find user that is trying to log in?

Reply:
Because our "custom" user extends AbstractUser model which extends AbstractBaseUser which has already defined password field, ss:
![image](https://user-images.githubusercontent.com/29740768/101993053-b3505480-3cb7-11eb-8063-a616c014b278.png)